samples/dotnet-svcutil.xmlserializer-instructions.md

```diff
@@ -0,0 +1,80 @@
+ # Using svctuil.xmlserializer on .NET Core
+
+Just like the svcutil XmlSerializer Type Generation function on desktop, dotnet-svcutil.xmlserializer NuGet package is the solution for .NET Core and .NET Standard Libraries. It pre-generates c# serialization code for the types used by Service Contract in the client applications that can be serialized using the XmlSerializer to improve the startup performance of Xml Serialization when serializing or de-serializing objects of those types using XmlSerializer. 
+
+You can start using the tool today following the instructions below. 
+
+## Prerequisites
+
+The following is required for svcutil.xmlserializer to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed.
+
+* [.NET Core SDK 2.1.2 or later](https://www.microsoft.com/net/download/windows)
+* [.NET Core runtime 2.1.0-preview1 or later](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.1.0-preview1-download.md)
+  
+## Instructions
+
+Here are the step by step instructions on how to use dotnet-svcutil.xmlserializer in a .NET Core console application.
+
+1. Create a WCF Service named 'MyWCFService' using default template 'WCF Service Application'.  Add [XmlSerializerFormat] attribute on the service method like the following
```

Comment:
I assume that this is with .NET Framework. You should clarify.

Reply:
OK